tools/GATK-VariantRecalibrator-Indels.cwl

```diff
@@ -17,13 +17,12 @@ doc: |
             -recalFile    $tmpDir/out.recal \
             -tranchesFile $tmpDir/out.tranches \
             -rscriptFile  $tmpDir/out.R \
-            -nt 4 \
```

Comment:
Apologies, I haven't noticed this one...
Good point! I think the parallelization part definitely goes as an issue for the time being. In my mind, and in terms of priorities, `JointGenotyping` should be addressed first because VQSR is already in place (and well, VQSR is only meaningful for reasonably large samples, and the logical way to call variants in such a case would be to do JointGenotyping of the called HC variants). I prefer addressing this one first, and tagging the parallelization part as an enhancement? Because, this part of enhancing the speed of processing could get hairy- for example, a user may wish to dedup/ BQSR/call variants per chromosome (or some other criteria), or skip the `PrintReads` stage in BQSR (a lot of improvement in speed results when you do this actually, and is worth the coding effort for the end user of the workflow) , or any variant there of...